### PR TITLE
Feat: Login-page Added

### PR DESCRIPTION
**Pull request by anurag-kkumarr**
Added login page 

- Email-password type of authentication
- Also added sign up button for new users
- Forgot button feature

Please review this PR

### CODE REVIEW

**Review by krgarima — COMMENTED** [review]
Some important suggestions on styling:
1. Improve color contrast ratio. It should between 4 and 8. Text are not clearly visible.
2. Also, work on choice of colors.
3. Elements are overlapping like 'forgot password' on login button.
4. Increase width of the search bar.

**Comment by krgarima** [comment]
Resolve merge conflicts.

**Review by rishichourasia — APPROVED** [review]
Few UI suggestions 
```
1 - Overall UI and color theming looks inconsistent 
2 -  login/signup Btn top and bottom padding looks unnecassary big 
3 - btn bg color doesn't match with your overall theme
4 - Resolve merge conflicts 
5 - In your home page add this line of code to your watch img element - 
object-fit: cover;
this will fit your img to the div element perfectly and not stretch

**Comment by anurag-kkumarr** [comment]
> Some important suggestions on styling:
> 
> 1. Improve color contrast ratio. It should between 4 and 8. Text are not clearly visible.
> 2. Also, work on choice of colors.
> 3. Elements are overlapping like 'forgot password' on login button.
> 4. Increase width of the search bar.

Made suggested changes. please look it once.

File: components/login-page/login.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+.pwd-info{
+    margin-right: 6rem;
+}
+
+.input-email, .input-pwd {
+    width: 15rem;
+    border: 1px solid var(--gray);
+    padding: var(--size-zero-five);
+    margin-bottom: var(--size-zero-two);
+}
```
Comment (krgarima) [inline components/login-page/login.css:40]: Add margin in .input-email
margin: 1rem auto;

File: components/login-page/login.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    width: 6rem;
+    font-size: var(--size-one-two);
+    border: 1px solid var(--black);
+    padding: var(--size-zero-five);
+    border-radius: var(--size-zero-two);
+    background-color: var(--color-primary);
+    color: var(--white);
+    cursor: pointer;
+    margin:0 auto var(--size-zero-four) auto;
+}
```
Comment (krgarima) [inline components/login-page/login.css:52]: Some margin in .login-in-btn like margin: 5px auto;

File: components/login-page/login.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+@import "/main.css";
+
+.login-layout {
+    display: flex;
+    flex-direction: column;
+    width: 40rem;
```
Comment (krgarima) [inline components/login-page/login.css:6]: Reduce width to 30rem.

File: components/login-page/login.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+.remember-me {
+    margin-right: 8rem;
+    color: var(--black);
+}
+.forgot-btn {
+    
+    margin-left: 10rem;
+    margin-bottom: var(--size-zero-five);
+    text-decoration: underline;
+    color: var(--color-primary);
```
Comment (krgarima) [inline components/login-page/login.css]: Choose a darker shade of this color for 'Forgot password'

**Comment by anurag-kkumarr** [comment]
Thanks for all the review

**Pull request merged**